apis/nucleus/src/components/listbox/ListBox.jsx

```diff
@@ -231,9 +237,11 @@ export default function ListBox({
     // If values have been filtered in the currently loaded page, we want to
     // prevent rendering empty rows by assigning the actual number of items to render
     // since count (qcy) does not reflect this in DQ mode currently.
-    const hasFilteredValues = ps.some((page) => page.qArea.qHeight < MINIMUM_BATCH_SIZE);
+    // If any qMatrix was not 100 length && result of it was less that qcy
+    // then => qTop + qHeight might indicate length of items as long as you scroll and fetch more items
+    const hasFilteredValues = ps.some((p) => p.qArea.qHeight !== MINIMUM_BATCH_SIZE);
     const h = Math.max(...ps.map((page) => page.qArea.qTop + page.qArea.qHeight));
```

Comment:
I suggest that 'h' should be changed to a new name which is easy to understand.

Reply:
I am hard in understanding `Math.max(...ps.map((page) => page.qArea.qTop + page.qArea.qHeight))`. Could you help me to explain it?